paasta_tools/paasta_maintenance.py

```diff
@@ -15,11 +15,21 @@
 import argparse
 import logging
 import sys
+import traceback
 from socket import getfqdn
+from socket import gethostbyname
 from socket import gethostname
 
 from paasta_tools import mesos_maintenance
-
+from paasta_tools import utils
+from paasta_tools.check_marathon_services_replication import load_smartstack_info_for_service
```

Comment:
Eh, let me do that on a second pass. That function and get_smartstack_replication_for_attribute and pretty closely related, so probably a whole slew of functions need to be moved into smartstack tools together.